App/populatePlaylist.js

```diff
@@ -0,0 +1,77 @@
+var playlistFromParse = JSON.parse(playlistsJSON);
+
+
+function unhidePlaylist(){
+
+    document.getElementById("playlist").style.visibility = "visible";
+    console.log(currentPlaylist);
+
+    var iDiv = document.createElement('div');
+    iDiv.id = 'content';
+    document.getElementById('content').appendChild(iDiv);
+
+
+//create header
+  for(var i = 0; i < currentPlaylist.songs.length; i++){
+
+      var innerDiv = document.createElement('div');
+      innerDiv.id = 'divSong';
+
+      innerDiv.style.width="600px";
+      innerDiv.style.height="100px";
+      innerDiv.style.backgroundColor= "white";
+      iDiv.appendChild(innerDiv);
+      var song = new Song(currentPlaylist.songs[i]);
+
+      var image = document.createElement('div');
```

Comment:
create separate methods/functions for each section: drawSongImage, drawSongTitle etc